frame/beefy-mmr/primitives/src/lib.rs

```diff
@@ -44,7 +44,7 @@ pub type Hash = [u8; 32];
 /// Generic hasher trait.
 ///
 /// Implement the function to support custom way of hashing data.
-/// The implementation must return a [Hash] type, so only 32-byte output hashes are supported.
+/// The implementation must return a [Hash](type@Hash) type, so only 32-byte output hashes are supported.
```

Comment:
what's this syntax?

Reply:
the type is ambiguous, with the Hash derive macro IIRC, and the compiler suggested this to disambiguate.

Reply:
https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html#namespaces-and-disambiguators